thoth/solver/python/python.py

```diff
@@ -317,6 +318,10 @@ def _do_resolve_index(python_bin, solver, all_solvers, requirements, exclude_pac
             )
             continue
 
+        # license solver
+        extracted_metadata["package_license"] = detect_license(extracted_metadata["importlib_metadata"]["metadata"])
+        print(extracted_metadata["package_license"])
```

Comment:
seems like debugging print is missed here
can you please change it to logging, if we should have it in source code.

Reply:
Ohh sorry, my bad. I forgot to change it after debugging :smiling_face_with_tear: